downloads.ini

```diff
@@ -77,9 +77,9 @@ output_path = third_party/ninja
 
 # Pre-built git
 [git]
-version = 2.23.0
-url =  https://github.com/git-for-windows/git/releases/download/v%(version)s.windows.1/PortableGit-%(version)s-64-bit.7z.exe
-download_filename = PortableGit-%(version)s-64-bit.7z.exe
-sha256 = 501d8be861ebb8694df3f47f1f673996b1d1672e12559d4a07fae7a2eca3afc7
+version = 2.24.0
+url =  https://github.com/git-for-windows/git/releases/download/v%(version)s.windows.2/PortableGit-%(version)s.2-64-bit.7z.exe
+download_filename = PortableGit-%(version)s.2-64-bit.7z.exe
```

Comment:
I think the `.2` should be moved to `version` because it's not a permanent feature of the filename.

Reply:
Oh, I see why this was done. Pretty strange versioning scheme. Nevermind.